src/installer/Directory.Build.props

```diff
@@ -138,175 +115,22 @@
 
   </PropertyGroup>
 
-  <PropertyGroup>
-    <TargetsWindows>false</TargetsWindows>
-    <TargetsOSX>false</TargetsOSX>
-    <TargetsiOS>false</TargetsiOS>
-    <TargetstvOS>false</TargetstvOS>
-    <TargetsAndroid>false</TargetsAndroid>
-    <TargetsLinux>false</TargetsLinux>
-    <TargetsUnix>false</TargetsUnix>
-    <TargetsUbuntu>false</TargetsUbuntu>
-    <TargetsLinuxMint>false</TargetsLinuxMint>
-    <TargetsDebian>false</TargetsDebian>
-    <TargetsRhel>false</TargetsRhel>
-    <TargetsOpensuse>false</TargetsOpensuse>
-    <TargetsFedora>false</TargetsFedora>
-    <TargetsCentos>false</TargetsCentos>
-    <TargetsOracle>false</TargetsOracle>
-    <TargetsSles>false</TargetsSles>
-    <TargetsBrowser>false</TargetsBrowser>
+  <PropertyGroup Condition="'$(PortableBuild)' == 'true'">
+    <HostMachineRidTargetsDebianPackages Condition="
+      $(NonPortableRuntimeOS.StartsWith('debian')) or
+      $(NonPortableRuntimeOS.StartsWith('ubuntu')) or
+      $(NonPortableRuntimeOS.StartsWith('linuxmint'))">true</HostMachineRidTargetsDebianPackages>
+    <!-- If the build machine isn't known to be Debian-based, try to build RPM packages. -->
+    <HostMachineRidTargetsRpmPackages Condition="'$(HostMachineRidTargetsDebianPackages)' != 'true'">true</HostMachineRidTargetsRpmPackages>
```

Comment:
The tooling that the installer build uses doesn't use these properties any more. If there's no explicit usage of these properties in the repo, you can delete this PropertyGroup